scripts/warn-outside-container

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+set -eu
+
+target="${1:-}"
+
+if [[ -z "${DISABLE_WARN_OUTSIDE_CONTAINER:-}" ]]; then
+    (
+        echo
+        echo
+        echo "WARNING: you are not in a container."
+        echo "Use \"make -f docker.Makefile $target\" or set"
+        echo "DISABLE_WARN_OUTSIDE_CONTAINER=1 to disable this warning."
+        echo
+        echo
+    ) >&2
+    sleep 5
```

Comment:
> Does it work with parallel installed? If it's not a requirement, perhaps we should detect, and skip parallel

It could work without parallel, but I don't see why we need to support running it without. We have a container that provides parallel, and we shouldn't support running it outside of a container.

`parallel` is only required for cross compiling from linux, so if someone is interested in building just a specific target they can run `script/build/osx`, `script/build/window` or `script/build/binary` which don't require parallel.